internal/bundle/testdata/fail-policy-missing-checks.mql.yaml

```diff
@@ -0,0 +1,189 @@
+policies:
```

Comment:
Most of the test data is not used, we should strip the test files to the specific test we are doing